mne/io/eyelink/tests/test_eyelink.py

```diff
@@ -201,6 +201,19 @@ def _simulate_eye_tracking_data(in_file, out_file):
         "SAMPLES\tPUPIL\tLEFT\tVEL\tRES\tHTARGET\tRATE\t1000.00"
         "\tTRACKING\tCR\tFILTER\t2\tINPUT"
     )
+
+    # Define your known BUTTON events
+    button_events = [
+        (5488529, 1, 0),
+        (5488532, 1, 1),
+        (5488540, 1, 0),
+        (5488543, 1, 1),
+        (5488550, 1, 0),
+        (5488553, 1, 1),
+        (5488571, 1, 0),
+    ]
```

Comment:
```suggestion
    # Define a few BUTTON press events
    button_events = [
        (7453400, 1, 0),
        (7453410, 1, 1),
        (7453420, 1, 0),
        (7453430, 1, 1),
        (7453440, 1, 0),
        (7453450, 1, 1),
        (7453460, 1, 0),
    ]
```

Reply:
I am just making sure that the timestamps here line up with the adjustments I made below.